test/dynamo/test_dynamo.py

```diff
@@ -267,9 +266,11 @@ def test_resnet18(self):
     # Graph 3: optimizer
     # Graph 4: sync input for backward
     # Graph 5: sync input for backward (TODO(JackCaoG) understand why there are two graphs)
-    self.assertEqual(met.metric_data('CompileTime')[0], 5)
+    # TODO @wonjoo CompileTime has increased 5->7 after enabling functionalization (#4680)
+    self.assertEqual(met.metric_data('CompileTime')[0], 7)
     # We execute 4 grphs per step when optimizer is enabled.
-    self.assertEqual(met.metric_data('ExecuteTime')[0], sample_count * 4)
+    # TODO @wonjoo ExecuteTime has increased by 1 after enabling functionalization (#4680)
+    self.assertEqual(met.metric_data('ExecuteTime')[0], (sample_count * 4) + 1)
```

Comment:
@JackCaoG, I just saw that this was failing as well -- the `ExecuteTime` is offset by 1 regardless of the sample size.